libnopegl/src/gen_specs.c

```diff
@@ -288,6 +288,11 @@ static int check_node_params(const struct node_class *cls)
     if (!par)
         return 0;
 
+    if (cls->params && !cls->opts_size) {
+        fprintf(stderr, "node type %s has parameters but no opts_size specified\n", cls->name);
```

Comment:
`but no opts_size specified` -> `but no opts_size is specified`

Reply:
Done